crates/pera-framework/packages/pera-system/sources/dwallet.move

```diff
@@ -56,10 +59,31 @@ module pera_system::dwallet {
         encryption_key: vector<u8>,
         key_owner_address: address,
         encryption_key_signature: vector<u8>,
+        key_owner_pubkey: vector<u8>,
     }
 
-    public(package) fun get_encryption_key(key: &EncryptionKey): vector<u8> {
-        key.encryption_key
+    /// An event emitted when an encryption key is created.
+    public struct CreatedEncryptionKeyEvent has copy, drop {
+        scheme: u8,
+        encryption_key: vector<u8>,
+        key_owner_address: address,
+        encryption_key_signature: vector<u8>,
+        key_owner_pubkey: vector<u8>,
+        session_id: ID,
+        encryption_key_id: ID,
+    }
+
+    /// An event emitted to start an encryption key verification process.
+    /// Since we cannot use native functions if we depend on Sui to hold our state,
+    /// we need to emit an event to start the verification process, like we start the other MPC processes.
```

Comment:
```suggestion
    /// An event emitted to start an encryption key verification process.
    /// Since we cannot use native functions if we depend on Sui to hold our state,
    /// we need to emit an event to start the verification process,
    /// like we start the other MPC processes.
```